### ISSUE TITLE
[RA] Let the combo box default to "including head-only rule" when there are two target words

### ISSUE THREAD

**Issue by bbryson** [rmlockwood/FLExTrans#791]
The new combo box is really nice, for the permutations!

For now it stays on "No permutations" when I have one or two target words.  It changes to "including head-only rule" when I have three.

With that as the default, it would be good to make that change once there are two words on the target side.

The permutations are working really well!  It's nice.

**Comment by rmlockwood** [comment]
The combo box gets enabled for me with just two words. @bbryson, make sure you have one of the two marked as head. I just verified this with 0.30.0.

**Comment by AndyBlack** [comment]
The default is now set to be with_head.  See https://drive.google.com/file/d/1CGXVvibkjl75T4CVkGuoDNAKa7IItmd-/view?usp=sharing.  The DTD changed, too.  The new one is at https://github.com/AndyBlack/ftrulegen/blob/master/FLExTransRuleGenerator.dtd

**Comment by rmlockwood** [comment]
@AndyBlack, we are going to reverse this. I talked with @bbryson and we decided that the default should be "no permutations". So the combo box would always start out as "no permutations" whether enabled or not. We don't think new users who do their first rule should get multiple rules ala permutations. It's something they should choose. With more user input over time we can see what users prefer.
One detail, though, let's make "including head-only rule" be before "omitting head-only rule". Thanks.
